django/test/client.py

```diff
@@ -30,7 +30,10 @@
 from django.utils.itercompat import is_iterable
 from django.utils.regex_helper import _lazy_re_compile
 
-__all__ = ('Client', 'RedirectCycleError', 'RequestFactory', 'encode_file', 'encode_multipart')
+__all__ = (
+    'AsyncClient', 'Client', 'RedirectCycleError', 'RequestFactory',
```

Comment:
I added also `AsyncRequestFactory`, we recommend imports from `django.test` so it's not really an issue, but we can add them.

Reply:
Which leads to another clean-up: some of these are not imported internally in Django from `django.test` ;)

Thanks for picking up on the missing pieces.

Reply:
It may not be possible to internally import all of from `django.test` because we can create circular imports.

Reply:
... but we can of course unify imports in docs and tests.

Reply:
Makes sense :+1: